src/AppBundle/Entity/Report/ProfDeputyManagementCost.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace AppBundle\Entity\Report;
+
+
+class ProfDeputyManagementCost
+{
+    /**
+     * ProfDeputyManagementCost constructor.
+     *
+     * @param $profDeputyManagementCostTypeId
+     * @param decimal $amount
+     * @param string  $hasMoreDetails
+     * @param string  $moreDetails
+     */
+    public function __construct($profDeputyManagementCostTypeId, $amount, $hasMoreDetails, $moreDetails)
+    {
+        $this->profDeputyManagementCostTypeId = $profDeputyManagementCostTypeId;
+        $this->amount = $amount;
+        $this->hasMoreDetails = $hasMoreDetails;
+        $this->moreDetails = $moreDetails;
+    }
+
+    /**
+     * @var decimal
+     *
+     * @JMS\Type("string")
+     * @JMS\Groups({"prof-deputy-management-costs"})
+     * @Assert\Type(type="numeric", message="profDeputyEstimateCost.amount.notNumeric", groups={"prof-deputy-estimate-costs"})
+     * @Assert\Range(min=0, max=100000000, minMessage = "profDeputyEstimateCost.amount.minMessage", maxMessage = "profDeputyEstimateCost.amount.maxMessage", groups={"prof-deputy-estimate-costs"})
+     */
+    private $amount;
```

Comment:
Same change as per api, this is the only field (renamed) and added to the trait

Reply:
Just to confirm, I'm dropping this class completely and just creating a property in the trait, something like `$generalManagementCostAmount`?

Reply:
Yes thats correct ^^ this can go then

Reply:
Great - dropped.